__init__.py

```diff
@@ -0,0 +1,9 @@
+import sublime_plugin
+from .event_loop import setup_event_loop, shutdown_event_loop
+
+setup_event_loop()
+
+
+class EventLoopListener(sublime_plugin.EventListener):
+    def on_exit(self) -> None:
+        shutdown_event_loop()
```

Comment:
the event loop starts, when the LSP module is loaded, 
and the loop ends when ST exits.